### ISSUE TITLE
 Verify slice projection theorem

### ISSUE THREAD

**Issue by Erhushenshou** [teamtomo/torch-fourier-slice#12]
I used your code to construct projections from 3D volume to verify the 3D slice projection theorm but failed. Have you tried the verificatio?

**Comment by alisterburt** [comment]
@Erhushenshou you have not specified what you tried or how you evaluated it...

**Comment by alisterburt** [comment]
yes, that is what is implemented here - for cubic volumes with even dimensions only

**Comment by McHaillet** [comment]
Closing as this is not an issue
